src/pkg/cli/debug.go

```diff
@@ -120,6 +120,24 @@ func findMatchingProjectFiles(project *types.Project, services []string) []*defa
 	return files
 }
 
+func IsProjectFile(basename string) bool {
+	return matchAny(patterns, basename)
+}
+
+func matchAny(patterns []string, name string) bool {
```

Comment:
I would suggest changing the name to matchFilePath or something along those lines. It's not obvious from the name that its not a generic pattern being matched but that of file paths.